src/NServiceBus.Transport.Msmq.Tests/APIApprovals.cs

```diff
@@ -9,7 +9,10 @@ public class APIApprovals
     [Test]
     public void Approve()
     {
-        var publicApi = typeof(MsmqTransport).Assembly.GeneratePublicApi();
+        var publicApi = typeof(MsmqTransport).Assembly.GeneratePublicApi(new ApiGeneratorOptions
+        {
+            ExcludeAttributes = new[] { "System.Runtime.Versioning.TargetFrameworkAttribute", "System.Reflection.AssemblyMetadataAttribute" }
```

Comment:
Let's not add the TargetFrameworkAttribute exclusion

Reply:
Yeah, it's only targetting .NET Framework.